modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerEpics.scala

```diff
@@ -113,9 +113,10 @@ abstract class TcsBaseControllerEpics[F[_]: {Async, Parallel, Logger}](
 
   import TcsBaseControllerEpics.*
 
+  private val mcsParkTimeout                  = FiniteDuration(60, SECONDS)
```

Comment:
By convention, we usually use TitleCase for constants (with capitalized first letter).